aj-may/eybc-bip
# Bip80 Permissioned edit feature for draft proposals

### Related Ticket



[BIP 80](https://eyblockchain.atlassian.net/browse/BIPT-80?atlOrigin=eyJpIjoiNTA4MWEwY2E2YTA5NGY5OWFjNTFmN2UxM2Y3OTAxMDkiLCJwIjoiaiJ9)

#### Dependent PR's

None

### Description

These changes allow only the author to make the changes to their draft. 

We do not show the "Edit" button on the row if the person is not the author for the proposal.
 
### QA Instructions



- Create a proposal as a draft with one wallet signed in. 
- Login with another wallet to check if the edit option is available. 

### Checklist



- [x] I have reviewed the code changes myself
- [x] My code meets all of the acceptance criteria of the ticket it closes.
- [x] I have removed unused code (e.g., `console.logs`, commented out blocks, etc.)
- [ ] I have made all necessary changes to the documentation.
- [x] Check PR title name is correct.
